content/en/user-guide/chaos-engineering/chaos-api/index.md

```diff
@@ -0,0 +1,195 @@
+---
+
+## Introduction
+
+LocalStack Chaos API allows you to mimic outages across any AWS region or service.
+Intentionally triggering service outages and monitoring the system's response in situations where the infrastructure is compromised offers a powerful way to test.
+This strategy helps gauge the effectiveness of the system's deployment procedures and its resilience against infrastructure disruptions, which is a key element of chaos engineering.
+
+You can use LocalStack Chaos API to cause API failures for following or any combination thereof:
+- Service
+- Region
+- Operation
+
+You can customise the HTTP error code and message that LocalStack responds with.
+If required, you can make the failures occur probabilistically.
+
+Furthermore, the Chaos API can also be configured to add a network latency for all calls.
+
+{{< alert title="Note">}}
+Chaos API is available as part of the LocalStack Enterprise plan.
+If you'd like to try it out, please [contact us](https://www.localstack.cloud/demo) to request access.
+{{< /alert >}}
+
+
+## Prerequisites
+
+The prerequisites for this guide are:
+
+- LocalStack Pro with [LocalStack CLI](https://docs.localstack.cloud/getting-started/installation/#localstack-cli) & [LocalStack Auth Token](https://docs.localstack.cloud/getting-started/auth-token/)
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- [Python](https://www.python.org/downloads/)
+
+
+## Configuration
+
+The disruption types supported by Chaos API are broadly categorised into two groups.
+**Service Faults** lead to an application-level HTTP error in an AWS service, and **Network Effects** introduce network-level effects to all connections.
+
+### Service Faults
+
+Service faults can be configured using the endpoint at `/_localstack/chaos/faults`.
+The configuration schema consists of an array of one or more rules, where each rule specifies the conditions for the fault to occur.
+When active, rules are evaluated sequentially on every request to LocalStack until the first match.
+
+The schema for the configuration is as follows.
+
+```json
+[
+    {
+        "region": "(str) Region name, e.g. 'ap-south-1'. If omitted, all regions are affected.",
+        "service": "(str) Name of the service, e.g. 'kinesis'. If omitted, all services are affected.",
+        "operation": "(str) Name of the operation, e.g. 'PutRecord'. If omitted, all operations are affected.",
+        "probability": "(num) Probability of invoking this rule, e.g. 0.5. If omitted, 1 is used.",
+        "error": {
+            "statusCode": "(int) HTTP status code to use in response, e.g. 503. If omitted, 503 is used.",
+            "code": "(str) Descriptive error code used in response. If omitted, 'ServiceUnavailable' is used."
+        }
+    },
+    ...
+]
+```
+
+The endpoint allows the following operations:
+- `GET`: Get current configuration
+- `POST`: Add new configuration
+- `PATCH`: Add a rule
```

Comment:
what is a configuration and what is a rule?

Reply:
I have defined this on line 46:

> Service faults can be configured using the endpoint at `/_localstack/chaos/faults`.
> The configuration schema consists of an array of one or more rules, where each rule specifies the conditions for the fault to occur.
> When active, rules are evaluated sequentially on every request to LocalStack until the first match.

Is it unclear?

Reply:
ah, now I see that the payload accepts an array. so this array would be the `configuration` and each element a `rule`? 

that would mean that a configuration is identified by an `id`, and `GET ./chaos/faults` would return the configs with their ids? To update a configuration I'd then `PATCH ./chaos/faults/<configuration_id>`

similarly for the `rules` there is a `DELETE` so I'd assume they are also identified by an id `DELETE ./chaos/faults/<fault_id>/<rule_id>

but maybe I'm thinking to complicated here... :thinking: having all that auto generated with make things much easier :D

Reply:
ok, then I guess just needs to be updated

> GET: Get current configuration
POST: Add new configuration
PATCH: Add a rule
DELETE: Delete a rule

maybe a GET and PUT is sufficient here then. instead of the term 'configuration' just 'list of rules' might be easier to understand.